### PR TITLE
Bump AMI/image IDs and fix bugs in preparation for next release

### PR DESCRIPTION
**Pull request by abyrne55**
This PR updates the default AMI IDs and Quay image IDs to their latest values per our [release documentation](https://github.com/openshift/osd-network-verifier/blob/main/RELEASE.md). AMI IDs were pulled from [Jenkins build 58](https://ci.int.devshift.net/job/gl-build-master-osd-network-verifier-golden-ami-packer/58/console).

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas
- ~~[ ] I have made corresponding changes to the documentation~~
- [x] I have made corresponding change to the default configuration files
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- [x] I have tested the functionality against ~~gcp~~ / aws, it doesn't cause any regression
- ~~[ ] I have added execution results to the PR's readme~~

## How to test this PR locally
I suggest running `make build`, `make test`, and integration tests for both "aws-classic" and  "aws-hcp" platform types. While the integration tests are running, ensure recently-added features (e.g., IMDSv2, no redirect following) are working as intended.

### CODE REVIEW

**Comment by abyrne55** [comment]
Fixed a couple bugs and passed both integration tests. Ready for review

/label tide/merge-method-squash

**Review by fahlmant — APPROVED** [review]
/lgtm
/approve
/hold

**Comment by abyrne55** [comment]
/approve
/unhold

@fahlmant please cut release v0.4.8 once this PR merges

**Pull request merged**
